reporting/sending_to_dhis2.md

```diff
@@ -0,0 +1,20 @@
+### Sending Reports to DHIS2
+
+1. Set the DHIS2 UUID for the facility that is sending the data by:
+
+   * Got to Legacy System Administration-&gt;Mantainance-&gt;Settings-&gt;Ugandaemr.
+     > ![](../assets/facilityUUID.PNG)
```

Comment:
@slubwama 

1. Add alt text between the square brackets in case the images do not load 
2. Change image names to be lower case (makes it easier not to make mistakes)